src/components/AccountScreen.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import {Button, Dialog, DialogActions, DialogContent, DialogContentText, Paper, TextField} from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+import CachedIcon from '@material-ui/icons/Cached';
+import './AccountScreen.css';
+
+
+const PASSWORD = "password"; // hardcoded password for testing the form
+
+
+const styles = {
+  button: {
+    margin: 5
+  }
+}
+
+class AccountScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "John Doe",
+      email: "johndoe@gmail.com",
```

Comment:
why are you initializing the name and email? Maybe in the input field you could just use placeholder instead?

Reply:
It only mimics the of how the user screen would function. 
This would be changed in later PR's after connecting to users table in DB and the form will be populated with currently logged in user's information.

Reply:
I see. Seems like I was confusing this with register

Reply:
I guess I should have been more specific about that in the description